tests/v2/validation/pipeline/rancherha/main.go

```diff
@@ -30,6 +30,8 @@ func main() {
 
 	configPackage := corral.PackagesConfig()
 
+	logrus.Info("Debug: Corral configs: ", corralConfig)
```

Comment:
do we want this still? Maybe `logrus.Debug` level?

Reply:
sorry, that was a missed log. I removed it, thats not needed.